src/components/PlaylistModal/PlaylistOption/index.jsx

```diff
@@ -64,8 +80,15 @@ const PlaylistOption = ({ video, playlist }) => {
 		} catch (error) {
 			showToast("Failed to delete playlist.", "error");
 		}
+
```

Comment:
Use of try  catch block is extensively done which is actually very nice for error handling.